content/getting-started/nextjs-guide.mdx

```diff
@@ -89,6 +89,7 @@ function MyApp({ Component, pageProps }) {
 }
 
 export default MyApp
+export {theme}
```

Comment:
Remove this and remove part and do export right where the const of the theme is defined.

export theme = extendTheme(...)